frontend/test/metabase/scenarios/collection_defaults.cy.spec.js

```diff
@@ -0,0 +1,148 @@
+import {
+  restore,
+  signInAsAdmin,
+  setupEmail,
+  signInAsNormalUser,
+} from "__support__/cypress";
+// Ported from initial_collection.e2e.spec.js
+
+// Z because the api lists them alphabetically by name, so it makes it easier to check
+const collection_name = "Z Collection";
+const sub_collection_name = "ZZ Sub-Collection";
+const pulse_name = "Test pulse";
+const dashboard_name = "Test Dashboard";
+
+function createPulse() {
+  cy.visit("/pulse/create");
+  cy.findByPlaceholderText("Important metrics").type(pulse_name);
+  cy.findByText("Select a question").click();
+  cy.findByText("Orders").click();
+  cy.findByPlaceholderText(
+    "Enter email addresses you'd like this data to go to",
+  )
+    .click()
+    .clear();
+  cy.contains("Bobby").click();
+  cy.findByText("To:").click();
+
+  cy.findByText("Robert Tableton").should("not.exist");
+  cy.findByText("Bobby Tables");
+  cy.findByText("Create pulse").click();
+}
+
+describe("scenarios > collection_defaults", () => {
+  before(() => {
+    // if (window.Cypress) {
+    //   event.preventDefault();
+    // }
+    restore();
+    cy.server();
```

Comment:
I had thought this was just used for stubbing requests. Is this leftover or used for something else?

Reply:
Yeah, this is leftover. Thanks for catching it!